content/en/docs/ambient/install/helm-installation/index.md

```diff
@@ -28,7 +28,21 @@ we encourage the use of Helm to install Istio for use in ambient mode. Helm help
 
 *See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
 
-## Install the components
+{{< gloss "ambient" >}}Ambient{{< /gloss >}} is made up of multiple components - for production deployments, it is strongly recommended to install the components separately to allow for more control during upgrades. Consult the [operations guidelines](/docs/ops/deployment) for production deployment considerations.
```

Comment:
```suggestion
Istio comprises many components (control plane, data plane, custom resources, etc). For production deployments, it is strongly recommended to install the components separately to allow for fine grained control during upgrades. Consult the [operations guidelines](/docs/ops/deployment) for production deployment considerations.
```

so is sidecar mode.  At least three of the charts are shared?

Does the operations docs actually talk about what you should do differently?  Is it a scaling suggestion? (You wouldn't scale istio-cni and I doubt you'd scale ztunnel)

Reply:
> so is sidecar mode. At least three of the charts are shared?

Yeah, this is sort of boilerplate. I'll enhance to mention that some components have different impacts on app traffic and node health than others (data plane vs control plane, really) and this impacts upgrade cadence considerations. 


> Does the operations docs actually talk about what you should do differently? Is it a scaling suggestion? (You wouldn't scale istio-cni and I doubt you'd scale ztunnel)

Scaling, and upgrade cadence.

Really, this should link to

https://preliminary.istio.io/latest/docs/ops/best-practices/deployment/

and that should be built out. I'll do that as part of this.

Reply:
Actually - what I was kinda hoping is that we could just link to wherever this https://github.com/istio/istio.io/pull/15247 ends up (under `/ops/` or wherever) as the "recommended/opinionated production setup".